sqlmesh/core/dialect.py

```diff
@@ -701,7 +703,16 @@ def pandas_to_sql(
 
 def normalize_model_name(table: str | exp.Table, dialect: DialectType = None) -> str:
     return exp.table_name(
-        normalize_identifiers(exp.to_table(table, dialect=dialect), dialect=dialect),
+        normalize_identifiers_func(exp.to_table(table, dialect=dialect), dialect=dialect),
+    )
+
+
+def normalize_identifiers(
+    value: str | exp.Expression, dialect: DialectType = None
+) -> exp.Expression:
+    return normalize_identifiers_func(
+        value if isinstance(value, exp.Expression) else exp.to_identifier(value),
+        dialect=dialect,
     )
```

Comment:
Do you think this is worth moving over to SQLGlot? The only difference with sqlglot's `normalize_identifiers` is that it knows how to handle strings, which was useful for validating the model's meta fields in this case.

Reply:
sure